packages/netdog/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "netdog"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-packages]
+source-groups = [
+    "netdog",
+    "dogtag",
```

Comment:
nit: we could put `dogtag` under `netdog` (like how various update-related crates are under `sources/updater`) but that can be done later